presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -228,7 +238,8 @@ public HiveMetadata(
             TypeTranslator typeTranslator,
             String prestoVersion,
             HiveStatisticsProvider hiveStatisticsProvider,
-            int maxPartitions)
+            int maxPartitions,
+            CollectibleStatisticsProvider collectibleStatisticsProvider)
```

Comment:
Let's move this above, to be after `HiveStatisticsProvider`

Reply:
`CollectibleStatisticsProvider` has been removed in favor of `ExtendedHiveMetastore#getSupportedColumnStatistics`.